addons/scopes/initKeybinds.inc.sqf

```diff
@@ -0,0 +1,95 @@
+["ACE3 Scope Adjustment", QGVAR(AdjustUpMinor), LLSTRING(AdjustUpMinor), {
+    // Conditions: canInteract
+    if !([ACE_player, objNull, ["isNotInside", "isNotSwimming"]] call EFUNC(common,canInteractWith)) exitWith {false};
+    // Conditions: specific
+    if (!([ACE_player] call CBA_fnc_canUseWeapon)) exitWith {false};
```

Comment:
I find it weird the above `if !(...)` is used sometimes, while this `if (!(...))` is used sometimes. Just a weird inconsistency.

Reply:
I am aware. I don't want to make formatting changes to this file, though admittedly I've changed `localize LSTRING` to `LLSTRING`. I want to reformat this component at a later date.